packages/@react-spectrum/combobox/test/ComboBox.test.js

```diff
@@ -5469,5 +5469,56 @@ describe('ComboBox', function () {
       expect(combobox).toHaveValue('');
       expect(listbox).not.toBeInTheDocument();
     });
+
+    it('supports RouterProvider', async () => {
+      let navigate = jest.fn();
+      let tree = render(
+        <Provider theme={theme} router={{navigate}}>
+          <ComboBox label="ComboBox with links">
+            <Item href="/one">One</Item>
+            <Item href="https://adobe.com">Two</Item>
+          </ComboBox>
+        </Provider>
+      );
+
+      let combobox = tree.getByRole('combobox');
+      let button = tree.getByRole('button');
+      triggerPress(button);
```

Comment:
if you already tried, don't worry about it, but we should try to use `await user.click` for any new tests